qa-pipelines/tasks/lib/cf-deploy-upgrade-common.sh

```diff
@@ -118,6 +118,12 @@ wait_for_release() {
         printf "%s pods are still pending after 80 minutes \n" "${release}"
         exit 1
     fi
+    if [[ ${cap_platform} == "eks" &&  ${namespace} == "scf" ]]; then
+        healthcheck_port=$(kubectl get svc tcp-router-tcp-router-public '-o=jsonpath={.spec.ports[?(@.name == "healthcheck")].port}' --namespace scf)
```

Comment:
`kubectl get service`.. especially because you're using the non-abbreviated form on line 124

Reply:
Also I would do `-o='...'` rather than `'-o=...'`